packages/fetchai/skills/aries_alice/handlers.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the handlers for the aries_alice skill."""
+
+import base64
+import binascii
+import json
+from typing import Dict, Optional, cast
+from urllib.parse import urlparse
+
+from aea.configurations.base import ProtocolId, PublicId
+from aea.mail.base import Envelope, EnvelopeContext
+from aea.protocols.base import Message
+from aea.protocols.default.message import DefaultMessage
+from aea.skills.base import Handler
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.protocols.http.serialization import HttpSerializer
+
+HTTP_CONNECTION_PUBLIC_ID = PublicId("fetchai", "http_client", "0.1.0")
```

Comment:
you can do import that from the package: `packages.fetchai.connections.http_client.connection import HTTP_CLIENT_CONNECTION_PUBLIC_ID`

Reply:
`HTTP_CLIENT_CONNECTION_PUBLIC_ID` does not exist.

Reply:
If you merge develop you can now do: `from packages.fetchai.connections.http_client.connection import PUBLIC_ID as HTTP_CLIENT_CONNECTION_PUBLIC_ID`

Reply:
Done